demo/src/components/SrlComponent.js

```diff
@@ -217,7 +217,7 @@ class SrlInput extends React.Component {
             {srlSentences.map((sentence, index) => {
               const selected = sentence === srlSentenceValue;
               return (
-                <option value={index} key={index} selected={selected}>{sentence}</option>
+                <option value={index} key={index} defaultValue={selected}>{sentence}</option>
```

Comment:
This will change what's selected initially, but mean selection defers to what's selected by the user after the fact.  Is that intentional?